components/four-oh-four-page/FourOhFourPage.style.js

```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components';
 import { H2, P } from '../../style/typography';
 import { max } from '../../lib/responsive';
+import { animated } from 'react-spring';
 
-const ImgBackground = styled.img`
+const ImgBackground = styled(animated.img)`
```

Comment:
Are you intending to mix the two libraries?

Reply:
Yep!